DSharpPlus.Commands/Invocation/ConverterUtil.cs

```diff
@@ -0,0 +1,8 @@
+namespace DSharpPlus.Commands.Invocation;
+
+/// <summary>
+/// Provides utility methods for handling converters and their state management.
+/// </summary>
+internal static class ConverterUtil
+{
+}
```

Comment:
Why does this exist?

Reply:
forgot to delete this when abandoning my attempt to fix more problems at once